### ISSUE TITLE
[BUG]: Dokapon Kingdom [NTSC/US] bad DMA/PATH3

### ISSUE THREAD

**Issue by refractionpcsx2** [PCSX2/pcsx2#10036]
### Describe the Bug

Game cycles through bad textures/colours during battles (possibly only on the US version)

Could be some weird DMA timing thing, or some strange PATH3 masking thing, not sure right now, unaffected by any settings.

![image](https://github.com/PCSX2/pcsx2/assets/6278726/708c7798-9486-4ca4-a8f8-f6f5db3a89f8)

### Reproduction Steps

Use the following savestate around version 1.7.5062, observe it screwing up.
[SLUS-21778 (CDE9832D).01.zip](https://github.com/PCSX2/pcsx2/files/12726064/SLUS-21778.CDE9832D.01.zip)

### Expected Behavior

Not breaking :P

### PCSX2 Revision

1.7.5062

### Operating System

Windows 11

### CPU

Ryzen 5900X

### GPU

RTX 2080Ti

### GS Settings

Happens in SW and HW, core problem (GS Dump tested on PS2)

### Emulation Settings

Tested every setting I could think of, nothing seems to really make any difference.

**Comment by Buzzardsoul** [comment]
I know this most likely is irrelevant, but I'd like to mention that this is an MTVU sensitive game. I'm hoping that this may help narrow down the issue, but I'm likely wrong in that regard.

**Comment by refractionpcsx2** [comment]
Yeah I've already messed with VU settings (and every setting available in the emulator), I had no luck.

**Comment by Buzzardsoul** [comment]
These R/W offsets might have something to do with this issue. 
Gonna add them in here just in case.

SLUS-21778

00BF5B14
00BF5B18
00BF5B1C
00BF5B20
00BF5B24
00BF5B28
00BF5B2C

**Comment by Goatman13** [comment]
Game work fine if I don't use your savestate. Maybe it's corrupted or something?

**Comment by Buzzardsoul** [comment]
Same as @Goatman13 here. Maybe it's a savestate from a person with a bad rom?

**Comment by refractionpcsx2** [comment]
No, it managed to go wrong for me too, I thought it was just the savestate too, as I tried the Japanese version too and it was fine, but then I tried the US version and I think I remember seeing it happen, maybe I was wrong, I can't reproduce it now.

Unrelated but there is actually a problem with the second map which is a regression from v1.7.4692, but gonna look in to that.

**Comment by KeoughDarklance** [comment]
I am having this same issue, with the textures going crazy. Has anyone figured out how to fix it yet?

**Comment by stehartt** [comment]
I am also experiencing this exact issue. It's quite severe and makes the game downright unplayable. The textures flash in multiple colors like crazy.

**Comment by Yuuta04** [comment]
Happened to me again on Nightly version v2.7.59, tried f9, changed to vulcan, still glitching is it safe file related? or anyway to solve this?
